catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/types/CoreAttributes.java

```diff
@@ -84,7 +84,7 @@
         descriptors.add(new AttributeDescriptorImpl(METADATA,
                 true /* indexed */,
                 true /* stored */,
-                true /* tokenized */,
+                false /* tokenized */,
```

Comment:
@pklinef  - this change (and the others you mention) were done to align with the attribute descriptor definitions in BasicTypes.  I suppose we could update these definitions in BasicTypes.